components/blitz/resources/omero/Constants.ice

```diff
@@ -296,6 +296,26 @@ module omero {
         MEANINTENSITY,
         SUMINTENSITY
       };
+
+      /**
+       * The projection axis
```

Comment:
This is an old commit (august) and might no longer be needed.
Thanks for pointing that out